site/data/roadmap.yaml

```diff
@@ -1,43 +1,54 @@
-- item: Release 2.16
+- item: Release 2.17
   status: ⌚ Planned
-  year: 2024
-- item: Import PCAP files
+  year: 2025
+  sponsor: Checkmarx Ltd.
+  sponsorlink: https://checkmarx.com/
+- item: Improved Authentication Handling
```

Comment:
Seems duplicate of "Continue improving authentication handling"

Reply:
Will review

Reply:
One is done and attributed to Jit, the other is on-going.

Reply:
Also it's what Simon asked for: https://github.com/zaproxy/zaproxy-website/pull/2932#issuecomment-2590482388

Reply:
The entry was removed in the meantime, was that on purpose? The CX sponsor changes were removed as well.

Reply:
Nope, I musta messed something up somehow ... grrr

Reply:
Still missing the sequence entry.

Reply:
Tks, fix coming